src/ajax.js

```diff
@@ -454,17 +454,19 @@ jQuery.extend( {
 
 				// Builds headers hashtable if needed
 				getResponseHeader: function( key ) {
-					var match;
+					var match, existingHeaderValues, headerValues;
```

Comment:
These new variables are expensive in terms of size increase to jquery.min.js gzipped size. I think we should leave them out.